extensions/simpread/src/sr.tsx

```diff
@@ -0,0 +1,190 @@
+    desc = unread.desc ? `> ${unread.desc}` : '',
+    note = unread.note ? `> ${unread.note}` : '',
+    template = `
+# ${unread.title}
+
+> 原文地址 [${host}](${unread.url}) at ${unread.create}
+
+${desc}
+
+${note}
+
+${tags.trim()}
+
+${annotes.trim().replace(/\*\*\*$/, '')}
+`;
+  return template;
+}
+
+export default function Command() {
+  const [state, setState] = useState<State>({});
+  const preferences: Preferences = getPreferenceValues();
+
+  useEffect(() => {
+    async function fetchUnrdist() {
+      try {
+        const favicon = 'favicon@default.png',
+          path = await getLocalStorageItem('simpread_config_path'),
+          config = readFileSync(!path ? preferences.path : path + '', 'utf8'),
+          unrdist = JSON.parse(config).unrdist,
+          items = unrdist
+            .map((unread: Unread) => {
+              let icon = unread.favicon;
+              const now = new Date(unread.create);
+              if (unread.favicon == undefined || unread.favicon == '') {
+                icon = favicon;
+              } else if (unread.favicon.startsWith('//res.wx.qq.com')) {
+                icon = 'https:' + unread.favicon;
+              } else if (unread.favicon.startsWith('/')) {
+                icon = favicon;
+              }
+              return {
+                id: unread.idx,
+                title: unread.title,
+                icon: icon,
+                lastModifiedAt: now,
+                desc: unread.desc,
+                url: unread.url,
+                markdown: getDetail(unread),
+              };
+            })
+            .map((info: Unread, idx: number) => (
+              <List.Item
+                key={idx.toString()}
+                title={info.title}
+                subtitle={info.desc}
+                icon={{ source: info.icon, mask: ImageMask.Circle }}
+                actions={
+                  <ActionPanel>
+                    <ActionPanel.Section>
+                      <OpenInBrowserAction title="Open Origin URL" url={info.url} />
+                      <PushAction
+                        title="Show Details"
+                        target={<Detail markdown={info.markdown} />}
+                        icon={{ source: 'sidebar-right-16' }}
+                      />
+                      <OpenInBrowserAction
+                        title="Open Local File"
+                        url={'http://localhost:7026/reading/' + info.id}
+                        icon={Icon.Desktop}
+                        shortcut={{ modifiers: ['cmd'], key: 'l' }}
+                      />
+                    </ActionPanel.Section>
+                  </ActionPanel>
+                }
+              />
+            ));
+        setState({ items: items });
+      } catch (error) {
+        setState({ error: new Error('Something went wrong') });
+      }
+    }
+    fetchUnrdist();
+  }, []);
+
+  if (state.error) {
+    showToast(ToastStyle.Failure, '获取失败，请确认是否存在 simpread_config.json');
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension